awx/ui/src/screens/WorkflowApproval/WorkflowApprovalList/WorkflowApprovalListItem.js

```diff
@@ -79,20 +79,14 @@ function WorkflowApprovalListItem({
         <ActionItem
           visible
           tooltip={
-            hasBeenActedOn
-              ? t`This workflow has already been ${status}`
-              : t`Approve`
+            hasBeenActedOn ? t`This has already been acted on` : t`Approve`
```

Comment:
we could possible move the status into parenthesis instead of removing altogether